programs/loft.sc

```diff
@@ -0,0 +1,356 @@
+__config() -> {
+    'stay_loaded'->true,
+    'commands'->{
+        'wand'->['wand'],
```

Comment:
the brackets here are not needed. you can ignore this comment tho, it's just an aesthetic thing.